### PR TITLE
Fix get_current_app_version to work in productive runtime

### PR DESCRIPTION
**Pull request by kaikae**
# Explain your changes

When running our project in prod runtime with a packaged release we noticed following error:
```
2024-03-26T07:48:53.916 app[48ed437a353ee8] fra [info] ** (UndefinedFunctionError) function Mix.Project.config/0 is undefined (module Mix.Project is not available)
2024-03-26T07:48:53.916 app[48ed437a353ee8] fra [info] Mix.Project.config()
2024-03-26T07:48:53.916 app[48ed437a353ee8] fra [info] (kinde_sdk 1.2.0) lib/kinde_sdk/sdk/utils.ex:83: KindeSDK.SDK.Utils.get_current_app_version/0
```
This is caused since Mix is not packaged into the release - we decided to fork and use another way to fetch the application version. 

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: mix.exs
Comment (kaikae) [inline mix.exs:46]: I had to update this dependency due to: https://github.com/deadtrickster/ssl_verify_fun.erl/pull/27
It most likely can be removed from the PR if the issue does not prevail for others.

**Comment by aliaamuqaddas** [comment]
LGTM

**Comment by kaikae** [comment]
@onderay why did you assign the PR to me? I do not have write access to the Repo so I can not merge it, in case you don't want this to be merged feel free to close the PR.

File: lib/kinde_sdk/sdk/utils.ex
Comment (kaikae) [reply lib/kinde_sdk/sdk/utils.ex:87]: hehe - nice catch! I didn't question what coderabbit suggested.

**Review by femz12 — APPROVED** [review]

**Comment by DanielRivers** [comment]
Thanks for your work here @kaikae and for reviewing @femz12. 

Will look at getting this deployed soon.

**Pull request merged**
